Documentation/docs-mobile/building-apps/build-items.md

```diff
@@ -127,7 +123,7 @@ with layout files:
 ## AndroidEnvironment
 
 Files with a Build action of `AndroidEnvironment` are used
-to [initialize environment variables and system properties during process startup](~/android/deploy-test/environment.md).
+to [initialize environment variables and system properties during process startup](/xamarin/android/deploy-test/environment).
```

Comment:
I don't know how the sync thing works, but this links to an old Xamarin doc?

So, we'd plan on moving this doc at some point?

Reply:
I didn't want to take the time to migrate over _every_ URL that mentioned `/xamarin`, and as the old `/xamarin/android` docs are only being *delisted* and not removed, I figured that this would be acceptable for now.

Mid-to-long term, *yes*, we should migrate over the referenced documentation so that everything lives within `/dotnet/android`.